### PR TITLE
First pull request by AlphaPwner

### PR DESCRIPTION
**Pull request by Kucharskov**
Team name: AlphaPwner
Our public key:
> -----BEGIN PUBLIC KEY-----
> MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0YHSBRBvWAR8ZlrEl5Lb
> KDp+O1TDiapuUM05jgiZoiRY1QzZyCWqKd0zcese9KAuoI01b5FlwrgWau360hmd
> OrqkNo1xjymwtX+WVX/RdlkONWZ6BXjKK31xsg2KousIaHD0ar8yvPGDWaWdOxYE
> p5n/PeRkVc7qCzTrG1RIrEWYKaEre8w8M6fg/JWpkboSAUZaLsLPC6EFEhK9u7mt
> oHAgU67Spi2b98r2kypy6V+cCMm6PXzpPvPp/ZAo9tY6hkM+cKN9HF56H/V1GLOA
> VLkf/oPJ+35vJon+uKknFS42o3H2ofohzArX5QmxwynlhqIDJ4x9qOPRO5OyKdU+
> IQIDAQAB
> -----END PUBLIC KEY-----

Happy christmas!

### CODE REVIEW

File: team_AlphaPwners/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# AlphaPwners Cookies
+### by team AlphaPwners
+
+```Python
+# Import elements
+import ingrediends from fridge as ing
+import microwave, bowl, utils from kitchen
+
+# Add ingredients into bowl
+bowl.add(ing.SUGAR, 4)
```
Comment (StevenVanAcker) [inline team_AlphaPwners/index.md]: Looks like you could use units here 😊

File: team_AlphaPwners/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+bowl.add(ing.CACAO, 2)
+bowl.add(ing.OIL, 1)
+bowl.add(ing.MILK, 3)
+bowl.add(ing.EGG, 1)
+bowl.add(ing.BAKING_POWDER, 0.25)
+bowl.add(ing.SALT, 0.05)
+
+# Mixing and baking
+bowl.mix_together()
+microwave.bake(bowl, 150)
```
Comment (StevenVanAcker) [inline team_AlphaPwners/index.md]: 150 watts? Minutes?

Comment (Kucharskov) [reply team_AlphaPwners/index.md]: You should check the kitchen.microwave docs :D Its fine or i really need to fix this. Some explainations are in film.

**Comment by Kucharskov** [comment]
I already do some fixes, now it should be fine without checking wideo or documentation :+1:

**Comment by StevenVanAcker** [comment]
Quick and easy! Just like I like it :D 

EGXc82rwqoZg2PSithlcz0G3gT/AOf3r4Vp3x6eKi3ROM3kJGPHOZa4fGTscT+iInU0aUpWJF9qT
NOC2DjstCLdysSdIX9530y6oqXLOW8cW7tKzfWz1DMRub+0YLLj9PFy7mwbDlXfzB9lbLgooQDs8
WLlgIIOoj9xxfpkjysRA2dFMfVZQHV/GQvydDc0XdBlgDe6Cy7Aoo9Yus7UvO2KF+nmAq0fgn+Yo
oy8RoqQ/jzFpqphh4ghGxGXejH4AyRcuK7X4Ozi4kiQrIH7j8tc905VIVsqiOYxc5h6SPryzOFdJ
Hs+/KrcbUnSUdbFjYA0uRI6gSNnDSb7C+oVILw==

**Pull request merged**